### DISCUSSION TITLE
Coordinates mapping from genome to transcriptome

### DISCUSSION THREAD

**Discussion by PengNi** [GoekeLab/xpore#104] [Q&A] [answered]
Dear the authors,

Thanks for developing this wonderful tool, it inspires me a lot!
 
Recently I want to benchmark xPore and other tools for RNA m6A identification using the data provided in the NBT paper. As described in the paper, xPore aligns DRS reads to the transcriptome, while m6A validation sets from m6ACE-seq/MiCLIP-seq provide genomic positions.

So as one genomic position may map mutliple transcriptome positions (_e.g._, chr1:1014146:+ to ENST00000624697.3:393:+, ENST00000624652.1:368:+, ENST00000379389.4:317:+), how do you convert site coordinates between genome and transcriptome? Do we need to choose the most abundant transcript aligned to each gene as described in the _Metagene analysis_ section of your paper, which in my view, by counting the number of reads (coverage) aligned to each transcript of each gene?

Thanks in advance!

Best,
Peng Ni

**Comment by ploy-np** [comment, marked as answer]
Hi @PengNi, 

I'm glad that it helps!

At the data preparation step, you can run `xpore dataprep` with `--genome` option to make the analysis (detection of differential RNA modification) based on genomic coordinates.

What is described in the metagene analysis section of the paper is that we ran `xpore diffmod` based on genomic coordinates; however, we also did the analysis of our results (modified positions) on transcriptomic functions, which resulted in Fig 2e. That's why we needed to choose the most abundant transcript aligned to each gene.
